### ISSUE TITLE
Infinite sliding doesn't slide in the right direction when there are only two images

### ISSUE THREAD

**Issue by vugman** [xiaolin/react-image-gallery#208]
I have a image gallery that has only two images.
When clicking on the right arrow on the second image the image slides right and not left.

**Comment by xiaolin** [comment]
fixed in version 0.8.4

**Comment by cocohub** [comment]
For anyone encountering this issue, I fixed it by passing the following function to the `onSlide` prop:
```
const handleSlide = (index: number) => {
  const numImages = imagesToDisplay.length;

  if (index === numImages - 1) {
    setCurrentIndex(0);
  } else {
    setCurrentIndex(index);
  }
};
```
No more weird sliding animations!
